gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -173,7 +173,8 @@ def add_arguments(self, parser):
     parser.add_argument(
         '--num_bigquery_write_shards',
         type=int, default=1,
-        help=('Before writing the final result to output BigQuery, the data is '
+        help=('Note: This flag is now deprecated and should not be used! '
```

Comment:
Can we nuke all the help text and just say "This flag is deprecated and may be removed in future releases" ?

Reply:
> Can we nuke all the help text and just say "This flag is deprecated and may be removed in future releases" ?

+1

Reply:
Done.